src/test/synapses.test.js

```diff
@@ -13,4 +13,35 @@ describe("Synapses", () => {
 
     publish("TEST_EVENT", { number: 10, string: "blue" });
   });
+
+  it("notifies all subscribers of an event", (done) => {
```

Comment:
We need one more test that reads last published event. Something like this:

```
publish("TEST_EVENT", { data: "shared" });

...

subscribe("TEST_EVENT", (result) => {
  expect(result).toMatchObject({ data: "shared" });
  done()
});
```

Reply:
i added a test using the map object since subscribe didnt behave that way.